src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporter.scala

```diff
@@ -149,6 +152,7 @@ class BlockImporter(
       ec: ExecutionContext
   ): Future[(List[Block], Option[Any])] =
     if (blocks.isEmpty) {
+      supervisor ! ProgressProtocol.ImportedBlock(importedBlocks.map(_.number).max)
```

Comment:
Minor: it could be `importedBlocks.headOption.map(_.number).getOrElse(0)`. In the case when we ex. try to import 1 block and it will be a duplicate `importedBlocks.map(_.number).max` will return an exception